### PR TITLE
KubeJS Powah Coolants Script

### PR DESCRIPTION
**Pull request by MagisterXero**
Added a script to allow the adding of solid coolants for Powah's Reactors
Please customize this as you'd like, these are really just reference points to use.

### CODE REVIEW

**Comment by MagisterXero** [comment]
Also to note, this script is required on the Client side in order for players to see the coolant list, and not have some weird glitchy stuff when inserting/removing coolant items from reactors.

File: kubejs/config/common.properties [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #KubeJS Common Properties
-#Sat May 01 23:14:14 CEST 2021
+#Sat Sept 28 21:45:14 CEST 2021
 exportVisualizerData=false
 packmode=default
 hideServerScriptErrors=false
 announceReload=true
 serverOnly=false
+invertClassLoader=true
```
Comment (NillerMedDild) [inline kubejs/config/common.properties:8]: This setting change worries me to be honest 😄 
@TemperedFool did you create something that relied on this being false?

**Review by NillerMedDild — CHANGES_REQUESTED** [review]

**Comment by NillerMedDild** [comment]
Looks good to me, thanks for your contribution! :D I just gotta make sure the settings change doesn't interfere with anything else before I merge, sit tight 😄

File: kubejs/config/common.properties (hunk 1 above)
Comment (NillerMedDild) [reply kubejs/config/common.properties:8]: I asked on the KubeJS Discord and am no longer worried 😄

**Pull request merged**